tests/foreman/maintain/test_clone.py

```diff
@@ -0,0 +1,166 @@
+"""Tests for satellite-maintain clone
+
+:Requirement: foreman-maintain
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: SatelliteClone
+
+:Assignee: lpramuk
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+import requests
+
+from robottelo import constants
+from robottelo.config import settings
+from robottelo.hosts import Satellite
+
+SSH_PASS = settings.server.ssh_password
+
+
+@pytest.mark.parametrize("sat_ready_rhel", [8], indirect=True)
```

Comment:
Instead of parametrizing, would suggest setting el8 default as similar to cores and memory in `sat_ready_rhel`?

Reply:
I want to use `sat_ready_rhel` fixture also for 6.11 if my tests are going to be merged to 6.11 branch.

Reply:
I would leave the parametrization here because with Sat 6.13 we will be moving to RHEL 9 as a platform. This parametrization would come in handy in this case.